### PR TITLE
Feature/add new contribution gh 297

### PR DESCRIPTION
**Pull request by kdv24**

### CODE REVIEW

File: app/src/Pages/AddContribution/AddContribution.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
```
Comment (andrewbiang888) [inline app/src/Pages/AddContribution/AddContribution.js]: I think we should move this code into the app/src/Pages/Portal/Contributions/Contributions.js page (so combining the two pages).

File: app/src/Pages/AddContribution/AddContribution.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  render() {
+    return (
+      <PageHoc>
+        <AddContributionForm {...this.props} />
+      </PageHoc>
+    );
+  }
+}
+
+export default connect(
```
Comment (andrewbiang888) [inline app/src/Pages/AddContribution/AddContribution.js]: When you combine the pages, this should be pulled out into `app/src/Pages/Portal/Contributions/index.js` and then do something like this:
```js
import { connect } from "react-redux";
import TopNavigation from "./Contributions";
import { isLoggedIn } from "../../state/ducks/auth";

export default connect(
state => { 
    return {state: state.auth}
  }, 
  dispatch => {
    return {
      login: (email,password) => dispatch(login(email,password)),
      dispatch
      }
    }
)(Contributions);

```

File: app/src/Pages/AddContribution/AddContribution.js (hunk 1 above)
Comment (andrewbiang888) [reply app/src/Pages/AddContribution/AddContribution.js]: Correction, it looks the pages you are creating should represent the path `/contributions/add` so maybe in that page. I would create the page here:
`/app/src/Pages/Portal/Contributions/Add` (we are trying to group the portal pages similar to how you would in Next.js)

**Review by andrewbiang888 — COMMENTED** [review]
Making good progress. Once I finish the Manage Portal section I will be able help with the `/contributions` tables!

File: app/src/components/Forms/AddContribution/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          <div>{formSections.dateOfContribution}</div>
+
+          <div>{formSections.typeOfContribution}</div>
+ 
+          {formSections.subTypeOfContribution} 
+          {formSections.typeOfContributor} 
+          {formSections.amountOfContribution} 
+          {formSections.oaeContributionType} 
+          {formSections.paymentMethod} 
+          {formSections.checkNumber}
```
Comment (jaronheard) [inline app/src/components/Forms/AddContribution/index.js]: I was imagining that the formSections would be a larger section, e.g. all of the above would be in `formSections.basic`. Then could be wrapped in a `div` that handles the layout of all of the fields within that section.

File: app/src/Pages/AddContribution/AddContribution.js (hunk 1 above)
Comment (kdv24) [reply app/src/Pages/AddContribution/AddContribution.js]: 👍

File: app/src/Pages/AddContribution/AddContribution.js (hunk 2 above)
Comment (kdv24) [reply app/src/Pages/AddContribution/AddContribution.js]: Ok, assuming `TopNavigation` should be `Contributions`.

File: app/src/components/Forms/AddContribution/index.js (hunk 3 above)
Comment (kdv24) [reply app/src/components/Forms/AddContribution/index.js]: 👍

**Comment by kdv24** [comment]
Thanks for the comments! Obviously I haven't done anything with styling at this point, but I have questions around that when I get there too.  I'm looking forward to seeing the Manage Portal section.

File: app/src/components/Forms/AddContribution/index.js (hunk 3 above)
Comment (kdv24) [reply app/src/components/Forms/AddContribution/index.js]: @jaronheard right now I have the  `header`, `basic`, `contributor`, and `other details` sections broken out into separate forms that are all pulled together to create the AddContribution page. I did it that way so I could break it all apart in my mind as I was trying understand the structure of the code, but one flaw as it sits right now is the `submit` button is probably only aware of the `header` form where it lives.  😬 

So, before I do any more that may be outside of what you're describing here, maybe we can talk tomorrow.  I'd also love to chat with you or maybe @andrewbiang888 about some styling questions.

**Pull request merged**